targets/the-build.xml

```diff
@@ -138,6 +138,7 @@
         <property name="drupal.site.database.database" value="${build.site}" />
         <property name="drupal.site.settings.file_public_path" value="sites/${drupal.site.dir}/files" />
         <property name="drupal.site.settings.file_private_path" value="../artifacts/private/${drupal.site.dir}" />
+        <property name="drupal.site.settings.file_temp_path" value="{$_ENV['TEMP']}" />
```

Comment:
I think that this `{$_ENV['TEMP']}` environment variable is specific to Acquia ([docs](https://docs.acquia.com/acquia-cloud/develop/env-variable/)).

Reply:
@becw should we add a specific `tmp` for every vendor (acquia / pantheon)?
Is there a better way to do it?

Reply:
Each vendor does their own handling of this, so I think that we can customize this behavior in the `settings.*.php` templates, rather than providing it as a setting (because then everybody would have to figure out how it should be set for whatever provider they're using). The temp directory is generally an absolute path, rather than a relative path like the public and private files directories, so it tends to be a bit more host-specific.

For example, Pantheon already handles the temp directory as part of its scaffolding: https://pantheon.io/docs/tmp

For Platform.sh, the temp directory is specified in the `.platform.app.yaml` file (`/tmp`), so we can use that.

Even the `settings.build.php` should be fine to hard-code it, since projects can edit that template as necessary (it's copied to each project at `.the-build/drupal/settings.build.php`).